step09/README.md

```diff
@@ -134,6 +134,6 @@ func getLivesAsEmoji() string{
 
 This version of `getLivesAsEmoji` will be less efficient than the version of the function that uses a buffer. Part of the reason for this performance difference is due to memory allocation required for the string concatenation, as we've seen before that strings in Go are immutable. 
 
-In the version of the function using the `+` operator, there is a memory allocation operation happening for every iteration of the for loop. While for the buffer version of the function there is only a single memory allocation happening when buffer is initiailized. A more detailed example of this performance difference is discussed [here](https://billglover.me/2019/03/13/learn-go-by-concatenating-strings/)
+In the version of the function using the `+` operator, there is a memory allocation operation happening for every iteration of the for loop. While for the buffer version of the function there is only a single memory allocation happening when buffer is initialized. A more detailed example of this performance difference is discussed [here](https://billglover.me/2019/03/13/learn-go-by-concatenating-strings/)
 
-[Take me to Next Step!](../stepxx/README.md)
\ No newline at end of file
+[Take me to Next Step!](../stepxx/README.md)
```

Comment:
As the step XX is not part of the tutorial please remove this line.

You could add the following instead:

```
## That's All Folks!

Congratulations! You've completed all the steps of the tutorial.

But your journey must not end here. If you are interested in contributing with a new step, have a look at the [TODO list](../TODO.md) or any open issues and submit a PR!
```